### PR TITLE
[Mobile UX] Style updates on buttons

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
#### What? Why?

Part of #4773 

Adjusts styling and alignment on cart and checkout buttons.

#### What should we test?

- Remove the left/right arrows in buttons -- they're taking up precious space in mobile
- Correct the positioning of the buttons in mobile so they fit in one single row
- Correct the positioning of the buttons in desktop so they're left / right aligned with the table
- Update the button colour to orange (because red indicates error)

#### Release notes

Updated styling on cart and checkout pages

Changelog Category: Changed

### CODE REVIEW

**Comment by Matt-Yorkley** [comment]
I added a little something here that wasn't in the spec: make the buttons on the cart page full-width if the screen is too small to fit them, instead of breaking them onto two lines in a disorderly way. It looks like this:

cart_buttons

**Comment by Matt-Yorkley** [comment]
@daniellemoorhead

**Comment by daniellemoorhead** [comment]
@yukoosawa see the additional functionality @Matt-Yorkley added to the screen, video screenshot displayed in comment above.

**Comment by yukoosawa** [comment]
@Matt-Yorkley @daniellemoorhead Yes additional functionality is fine. Ship it!

File: app/assets/stylesheets/darkswarm/branding.css.scss [hunk 1]
```diff
@@ -54,6 +54,7 @@ $teal-400: #4cb5c5;
 $teal-500: #0096ad;
 
 $orange-400: #ff9466;
+$orange-450: #f4704c;
```
Comment (sauloperez) [inline app/assets/stylesheets/darkswarm/branding.css.scss:57]: :trollface:

File: app/assets/stylesheets/darkswarm/shopping-cart.css.scss [hunk 2]
```diff
@@ -115,3 +115,14 @@
     height: 36px;
   }
 }
+
+.links {
+  .button {
+    padding: 1.125rem 0 1.1875rem;
+    width: 210px;
+
+    @media all and (max-width: 480px) {
```
Comment (sauloperez) [inline app/assets/stylesheets/darkswarm/shopping-cart.css.scss:124]: :see_no_evil: we don't have the responsive breakpoints defined in a var?

**Comment by daniellemoorhead** [comment]
**Testing Notes:** GOOD TO GO!

- [x] Remove the left/right arrows in buttons -- they're taking up precious space in mobile.
Screen Shot 2020-04-23 at 7 54 28 pm

- [x] Correct the positioning of the buttons in mobile so they fit in one single row
IMG_6056

- [x] Correct the positioning of the buttons in desktop so they're left / right aligned with the table.
Screen Shot 2020-04-23 at 8 03 02 pm

- [x] Update the button colour to orange (because red indicates error)

And bonus: the buttons on the view cart dropdown and the product list are orange 🎉 
Screen Shot 2020-04-23 at 7 52 17 pm

Full steam ahead @Matt-Yorkley

**Pull request merged**